### PR TITLE
Allow to set various Redis parameters

### PR DESCRIPTION
**Pull request by AymericDu**
##### SUMMARY

Allow to set various Redis parameters

Jira: OB-461

##### ISSUE TYPE

 - Bugfix Pull Request

##### COMPONENT NAME

- `account`

##### SDS VERSION

```
openio 4.7.2.dev19
```

### CODE REVIEW

File: oio/common/redis_conn.py [hunk 1]
```diff
@@ -16,41 +16,103 @@
...
+
+    @property
+    def conn_slave(self):
+        """Retrieve Redis connection (normal or sentinel)"""
+        if self._sentinel:
+            return self._sentinel.master_for(self._sentinel_name)
+        if not self._conn:
+            self._conn = self.__redis_mod.StrictRedis(
+                host=self._host, port=self._port,
+                **self._conn_kwargs)
```
Comment (fvennetier) [inline oio/common/redis_conn.py]: I would return `self.conn` to avoid duplicating these 4 lines.

File: oio/common/redis_conn.py [hunk 2]
```diff
@@ -16,41 +16,99 @@
...
         return self._conn
 
+    @property
+    def conn_slave(self):
+        """Retrieve Redis connection (normal or sentinel)"""
+        if self._sentinel:
+            return self._sentinel.master_for(self._sentinel_name)
```
Comment (murlock) [inline oio/common/redis_conn.py]: IMP it should be `slave_for`

File: oio/container/backup.py [hunk 3]
```diff
@@ -794,7 +795,25 @@ def __init__(self, conf):
         self.REDIS_TIMEOUT = self.conf.get("redis_cache_timeout",
                                            self.REDIS_TIMEOUT)
 
-        super(ContainerBackup, self).__init__(self.conf)
+        redis_conf = {k[6:]: v for k, v in self.conf.items()
```
Comment (murlock) [inline oio/container/backup.py]: Maybe we should factorize code to extract redis paramters

File: oio/common/redis_conn.py (hunk 2 above)
Comment (AymericDu) [reply oio/common/redis_conn.py]: Oops

**Pull request merged**